design/data-model.md

```diff
@@ -0,0 +1,47 @@
+# Rook Data Model
+
+
+```
+# Operator
+The Rook operator is a single instance in Kubernetes
+The operator manages multiple Rook storage clusters
+The operator manages all CRDs for the Rook clusters
+
+# Storage Cluster
+The cluster CRD defines desired settings for a storage cluster
+All resources for a Rook cluster are created in the same Kubernetes namespace
+A cluster has three mons
```

Comment:
is this true for the recent affinity/anti-affinity changes?

the number of monitors should be able to scale up with cluster size, right?

Reply:
It is still true that there are three monitors. I'm anticipating the number of mons would be a setting on the cluster.yaml, but for now it's exactly three. Since that might change, I'll rephrase this line